website/views/shared/tasks/meta_controls.jade

```diff
@@ -45,7 +45,7 @@
         span.glyphicon.glyphicon-bullhorn(tooltip=env.t('challenge'))
         | &nbsp;
     // delete
-    a(ng-if='!task.challenge.id || obj.leader._id === User.user._id', ng-click='removeTask(task, obj)', tooltip=env.t('delete'))
+    a(ng-if='!task.challenge.id || obj.leader._id === User.user._id || user.contributor.admin', ng-click='removeTask(task, obj)', tooltip=env.t('delete'))
```

Comment:
Not a big deal, but this is probably more clear if moved to a function in the controller called `canEditTask`.